pandas/core/common.py

```diff
@@ -479,6 +481,41 @@ def _dict_compat(d):
                 for key, value in iteritems(d))
 
 
+def standardize_mapping(into):
+    """
+    Helper function to standardize a supplied mapping.
+    .. versionadded:: 0.21.0
+
+    Parameters
+    ----------
+    into : instance or subclass of collections.Mapping
+        Must be a class, an initialized collections.defaultdict,
+        or an empty instance of a collections.Mapping subclass.
```

Comment:
I don't think this last statement is true, it *doesn't* have to be empty (nor should we require it)